pybossa/api/task.py

```diff
@@ -89,6 +89,13 @@ def _preprocess_post_data(self, data):
         if 'n_answers' not in data:
             project = project_repo.get(project_id)
             data['n_answers'] = project.get_default_n_answers()
+        if 'user_pref' in data and 'languages' in data['user_pref']:
```

Comment:
Very minor. I would write in the following way. Think it might be more concise and robust:
```
user_pref = data.get('user_pref', {})
if user_pref.get('language'):
  user_pref['language'] = [s.lower() for s in user_pref.get('language', [])]
if user_pref.get('location'):
  user_pref['location'] = [s.lower() for s in user_pref.get('location', [])]
if user_pref.get('email'):
  user_pref['email'] = [s.lower() for s in user_pref.get('email', [])]
 ```

Reply:
Thanks Xi, I think this is more concise

Reply:
The `user_pref` field uses keyword "locations" instead of "location".